frontend/src/app/auth/oauth2.service.ts

```diff
@@ -12,6 +12,10 @@ import { LocalStorageService } from '../core/services/local-storage/local-storag
     providedIn: 'root'
 })
 export class OAuth2Service {
+    private signedin = new BehaviorSubject<boolean>(false);
```

Comment:
What's this BehaviorSubject being used for? I'm not against it, I'm just not sure why it was added.